src/cmds/farm.py

```diff
@@ -0,0 +1,321 @@
+import click
+import sys
+import time
+import math
+from datetime import datetime
+from typing import Tuple, Optional, Callable, List, Dict, Any
+
+import aiohttp
+import asyncio
+
+from src.rpc.wallet_rpc_client import WalletRpcClient
+from src.rpc.harvester_rpc_client import HarvesterRpcClient
+from src.rpc.full_node_rpc_client import FullNodeRpcClient
+from src.rpc.farmer_rpc_client import FarmerRpcClient
+from src.util.bech32m import encode_puzzle_hash
+from src.util.byte_types import hexstr_to_bytes
+from src.util.config import load_config
+from src.util.default_root import DEFAULT_ROOT_PATH
+from src.util.ints import uint64, uint16
+from src.cmds.units import units
+from decimal import Decimal
+from src.wallet.util.transaction_type import TransactionType
+
+MINUTES_PER_BLOCK = (24 * 60) / 4608  # 0.3125
+
+
+def compute_wallets_stats(wallets):
```

Comment:
This is based on what I found in the webapp.

Reply:
+1